packages/flutter/test/services/text_input_test.dart

```diff
@@ -140,12 +140,28 @@ void main() {
       const TextInputType decimal = TextInputType.numberWithOptions(decimal: true);
       const TextInputType signedDecimal =
         TextInputType.numberWithOptions(signed: true, decimal: true);
+      const TextInputType multiline = TextInputType.multiline;
```

Comment:
you don't need to create constants for values that are just the existing constant.